plenario/models.py

```diff
@@ -1,23 +1,21 @@
-from uuid import uuid4
-from datetime import datetime
-
-from sqlalchemy import Column, String, Boolean, Date, DateTime, \
-    Text, func, Table, select, Integer
-from sqlalchemy.dialects.postgresql import ARRAY, JSONB
-from geoalchemy2 import Geometry
-from sqlalchemy.orm import synonym
+import json
 import sqlalchemy as sa
-from plenario.utils.helpers import get_size_in_degrees
+
+from collections import namedtuple
+from datetime import datetime
 from flask_bcrypt import Bcrypt
-from itertools import groupby
-import json
+from geoalchemy2 import Geometry
 from hashlib import md5
+from itertools import groupby
 from operator import itemgetter
+from sqlalchemy import Column, String, Boolean, Date, DateTime, Text, func
+from sqlalchemy import Table, select, Integer
+from sqlalchemy.dialects.postgresql import ARRAY, JSONB
+from sqlalchemy.orm import synonym
+from uuid import uuid4
 
 from plenario.database import session, Base
-from plenario.utils.helpers import slugify
-
-from collections import namedtuple
+from plenario.utils.helpers import get_size_in_degrees, slugify
```

Comment:
just rearranging the imports -- "a foolish consistency..." I know, but sometimes I just have urges to alphabetize